src/plugin/fieldViews/DropdownFieldView.ts

```diff
@@ -1,13 +1,28 @@
 import type { Node } from "prosemirror-model";
 import type { EditorView } from "prosemirror-view";
-import type { Option } from "../types/Element";
 import { AttributeFieldView } from "./AttributeFieldView";
+import type { BaseFieldSpec } from "./FieldView";
+
+type Option = { text: string; value: string };
```

Comment:
We've lost the type parameter here which allowed value to represent a variety of different types. Was this intentional?

Reply:
Yes, and we should have added a note – IIUC an option selected in a dropdown will always we of type `string`, I don't think anything else was possible in practice.

Reply:
I'll update the dev notes in the PR.

Reply:
It was serialising the value as JSON, then parsing it out to support this. Admittedly it might be unnecessary but should work.

Reply:
Oh, wow, I missed this! As an aside, surely that sort of behaviour should be up to the consumer? It seems odd to bake that assumption into the core library, as a string type is IIUC by far the most likely choice for a consumer. (I don't recall ever seeing a dropdown that encoded JSON into its values. Perhaps I've just ... never truly lived 😁 )

We'll restore this behaviour in the meantime, as if we do want to make a change that should be a separate PR.

Reply:
That's fair to say. It seemed like a reasonable idea at the time, incase you wanted the dropdown to represent different values. Possibly I didn't stop to question it enough.